common.sh

```diff
@@ -49,8 +49,8 @@ NODES_PLATFORM=${NODES_PLATFORM:-"libvirt"}
 MASTER_NODES_FILE=${MASTER_NODES_FILE:-"ocp/master_nodes.json"}
 
 export RHCOS_IMAGE_URL=${RHCOS_IMAGE_URL:-"https://releases-rhcos.svc.ci.openshift.org/storage/releases/ootpa/"}
-export RHCOS_LATEST="$(curl ${RHCOS_IMAGE_URL}/builds.json | jq -r ".builds[0]")"
-export RHCOS_IMAGE_VERSION="${RHCOS_IMAGE_VERSION:-${RHCOS_LATEST}}"
+export RHCOS_INSTALLER_PIN=$(jq -r '.buildid' $GOPATH/src/github.com/openshift-metalkube/kni-installer/data/data/rhcos.json)
```

Comment:
There's no `eval $(go env)` line in 04, so $GOPATH doesn't set.
We should probably move the `go env` into common.sh if we're going to rely on it here.

Reply:
Ah yeah I just said the same above before seeing this :)